### PR TITLE
New grid

### PR DESCRIPTION
**Pull request by LalightKosh**
feat: made a grid section

### CODE REVIEW

File: src/components/Lead/Lead.module.css [hunk 1]
```diff
@@ -97,6 +97,10 @@
         margin-right: auto;
         margin-left: 24px;
     }
+
+    .caption {
```
Comment (totfront) [inline src/components/Lead/Lead.module.css:101]: this change does not relevant to the grid section and should be moved to another PR, but you can leave it here if you do not want to create another one. Personally, I do not like it, but we did not have this requirement in the beginning of the project.

**Comment by totfront** [comment]
Critical issues:

1. build is failing (It's getting annoying). Y.Practicum would decline the review without hesitations and companies will decline your test projects for them too if they can not run it.
2. there is a wrong gap on the largest screens.

Optional improvements:

1. This is a section with a list of images. So the HTML should look like `<section><ul><li></li><li></li></ul></section>`
2. Naming `photo_1` is acceptable, but when you already have a description in `alt` for it I would use it for file and variable naming as well. For example "roadSignPhoto"
3. "/images/photogrid" in the files stricture should be "/images/photoGrid". Do not forget to run the project and clean the paths where needed if you will fix it

**Comment by LalightKosh** [comment]
>build is failing (It's getting annoying). Y.Practicum would decline the review without hesitations and companies will decline your test projects for them too if they can not run it.

**solved on the call**

> 2. there is a wrong gap on the largest screens.

**It is 16/16, both in my project and Figma**

> Optional improvements:

> 1. This is a section with a list of images. So the HTML should look like `<section><ul><li></li><li></li></ul></section>`

**Done**

> 2. Naming `photo_1` is acceptable, but when you already have a description in `alt` for it I would use it for file and variable naming as well. For example "roadSignPhoto"

**Done**

> 3. "/images/photogrid" in the files stricture should be "/images/photoGrid". Do not forget to run the project and clean the paths where needed if you will fix it

to be honest I am a bit afraid of changing capitalization :)

**Review by totfront — APPROVED** [review]

File: src/components/Lead/Lead.module.css [hunk 2]
```diff
@@ -114,6 +118,10 @@
         margin-top: 8px;
         margin-left: 48px;
     }
+
+    .caption {
```
Comment (totfront) [inline src/components/Lead/Lead.module.css:122]: This change is irrelevant to the grid section and should be moved to another PR, but you can leave it here if you do not want to create another one. Personally, I do not like it, but we did not have this requirement at the beginning of the project.

File: src/components/Lead/Lead.module.css [hunk 3]
```diff
@@ -126,4 +134,8 @@
     .caption {
         margin-left: 148px;
     }
+
+    .caption {
```
Comment (totfront) [inline src/components/Lead/Lead.module.css:138]: This change is irrelevant to the grid section and should be moved to another PR, but you can leave it here if you do not want to create another one. Personally, I do not like it, but we did not have this requirement at the beginning of the project.

File: src/components/PhotoGrid/PhotoGrid.js [hunk 4]
```diff
@@ -1,44 +1,82 @@
...
-        className={styles.photo}
-        alt="Mountainous forest"
-      />
-      <img src={gridPhoto_11} className={styles.photo} alt="Cliffs" />
-      <img
-        src={gridPhoto_12}
-        className={styles.photo}
-        alt="Cloudy mountain tops"
-      />
+    <section className={styles.section}>
```
Comment (totfront) [inline src/components/PhotoGrid/PhotoGrid.js:17]: Cool semantic usage in here 😍

**Review by totfront — APPROVED** [review]
LGTM ✨
